### PR TITLE
Update Albania generation capacity

### PR DESCRIPTION
**Pull request by autipial**
This PR updates Albania generation capacity based on ERE (energy regulator of Albania) data.
Related to #4180.

### CODE REVIEW

File: DATA_SOURCES.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ For many European countries, data is available from [ENTSO-E](https://transparen
   - Wind: [Finnish Wind Power Association](https://tuulivoimayhdistys.fi/en/wind-power-in-finland/map 'Note: the map states 43MW for the Långnabba project, but as [Vestas](https://www.vestas.com/en/media/company-news/2020/vestas-develops-highly-customised-power-plant-solution--c3249491) and [Kraftnat](https://kraftnat.ax/) state 42MW, the lower figure was used')
   - Solar: [Ålands statistik- och utredningsbyrå](https://www.asub.ax/sv/ekologisk-hallbarhet-17-installerad-kapacitet-elproduktion-solpaneler-pa-aland)
   - Oil: [Kraftnät Åland](https://kraftnat.ax/stamnatet/)
-- Albania: [IRENA](https://www.irena.org/publications/2022/Apr/Renewable-Capacity-Statistics-2022)
+- Albania: [ERE](https://ere.gov.al/images/2022/04/Raporti%20Vjetor%202021%20perfundimtare.pdf)
```
Comment (VIKTORVAV99) [inline DATA_SOURCES.md:171]:
```suggestion
- Albania:
  - Biomass: [IRENA](https://www.irena.org/publications/2022/Apr/Renewable-Capacity-Statistics-2022)
  - Other: [ERE](https://ere.gov.al/images/2022/04/Raporti%20Vjetor%202021%20perfundimtare.pdf)
```

File: config/zones.json [hunk 2]
```diff
@@ -42,11 +42,11 @@
       [21.0200403175, 42.6882473822]
     ],
     "capacity": {
-      "biomass": 1,
-      "hydro": 2289,
-      "solar": 22
+      "gas": 98,
+      "hydro": 2484,
+      "solar": 23
```
Comment (VIKTORVAV99) [inline config/zones.json:47]:
```suggestion
      "biomass": 1,
      "gas": 98,
      "hydro": 2484,
      "solar": 23
```

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]
Biomass capacity should be kept and IRENA should still be the source for it.

**Comment by autipial** [comment]
I am very surprised by the reference from IRENA to this 1MW of biomass/bioenergy capacity, since I never saw it mentioned in any document related to the electricity grid of Albania (especially in the documents published by Albania's electricity producers / transmitter / distributor / regulator).
Even IRENA itself doesn't mention it in its [assessment of Albania's renewable capacities](https://www.irena.org/publications/2021/March/Renewables-Readiness-Assessment-Albania) in 2021.

Of course if you think it is best to keep it, I will amend the PR.

BTW I realize that I also made a mistake regarding the thermal power plant of Vlorë -- it is designed to be able to burn gas, but its primary combustible is oil. I will change the PR accordingly.

**Comment by VIKTORVAV99** [comment]
> https://www.irena.org/publications/2022/Apr/Renewable-Capacity-Statistics-2022

Technically it's not biomass per say but renewable municipal waste but currently electricityMap treats waste as biomass so that is where the 1 MW comes from.

**Review by VIKTORVAV99 — APPROVED** [review]

**Pull request merged**